src/main/java/ToDo.java

```diff
@@ -0,0 +1,13 @@
+public class ToDo extends Task {
+    protected String description;
+    protected boolean isDone;
+
+    public ToDo(String description) {
+        super(description);
+    }
+
+    @Override
+    public String toString() {
+        return "[T][ " + super.getStatusIcon() + " ] " + super.getDescription();
```

Comment:
perhaps, you can use super.toString() to replace super.getStatusIcon() and super.getDescription()